CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Added Makefile to simplify building and developing the project locally. (#67)
 
+- Updated `package-lock.json` to lockfile version 2 with `npm install`, using
+  npm@7.24.1. (#74)
+
```

Comment:
Nodejs v14 still ships with npm v6, so updating only NPM will mean some devs will use lockfile v1 and some v2.

Is the plan to force all devs to manually update npm?

We could not update to Nodejs v16 before:

> Turns out that [Node v16 is not supported by Angular yet](https://github.com/angular/angular-cli/issues/20757#issuecomment-840485441) and [there are apparently some bugs somewhere in the dependency tree](https://github.com/angular/angular-cli/issues/19957#issuecomment-775407654) which led the Angular team to start enforcing node v14 or v16 ([see `engines` property in their `package.json`](https://unpkg.com/browse/@angular/cli@12.0.2/package.json)).
>
> (Source: #27)

Even the latest version of Angular (v12.2.7) has this `engines` setting to enforce Nodejs v14 (https://unpkg.com/browse/@angular/cli@12.2.7/package.json)

Reply:
Oh, I see. Now that you mention it I do remember reading that.

I found this https://github.com/angular/angular-cli/issues/19957#issuecomment-786177315 which says that `12.0.0-next.2 or greater` supports NPM v7 now at least, but unsure if the benefits of using NPM v7 warrants forcing people to manually update.

I'm leaning more towards just downgrading my own version, as upgrading to NPM v7 was something I most likely did while working on something else, and I've had no issue using NPM v6 for wharf.

Reply:
If you can build and run the project using Nodejs 16 then go for it